src/main/java/uk/ac/ebi/ampt2d/accession/serial/block/MonotonicRangePriorityQueue.java

```diff
@@ -0,0 +1,68 @@
+/*
+ *
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ampt2d.accession.serial.block;
+
+import java.util.PriorityQueue;
+
+public class MonotonicRangePriorityQueue extends PriorityQueue<MonotonicRange> {
+
+    private long totalOfValuesInQueue;
```

Comment:
`numQueuedValues` is closer to our usual naming (and `totalOfValues` is not very idiomatic anyway :stuck_out_tongue_winking_eye: )

Reply:
Not happy with the original or the alternative :/